tasks/main.yml

```diff
@@ -1,13 +1,5 @@
 ---
 
-- name: Install package
-  become: True
-  apt:
-    name:
-      - kubernetes-hyperkube-kube-proxy
-      - socat
```

Comment:
Are you sure we don't need socat anymore?

Reply:
not that we don't need it but it is already a dependency of `kubernetes-node`, so it is still installed.